projects/ACore/test/ExceptionTest.cpp

```diff
@@ -0,0 +1,60 @@
+#include "pch.hpp"
+
+#include <Catch2/catch.hpp>
+
+namespace exceptiontest
+{
+TEST_CASE("[acore::Exception]")
+{
+#ifndef _WIN32
```

Comment:
MSVC `std::string` is standard layour now, the `ifdef` should no longer be needed.